apps/server/src/app/export.controller.ts

```diff
@@ -41,7 +42,7 @@ export class ExportController extends BaseController {
       let validationRequests = [];
 
       const validateResource = (resource: DomainResource) => {
-        return new Promise((resolve, reject) => {
+        return new Promise((innerResolve, innerReject) => {
```

Comment:
let's change these back to "resolve, reject" instead of "innerResolve, innerReject". "inner" has lost its value now that we've changed direction on the fix.

Reply:
just as some background so it doesn't seem random, since the code snippet here in git doesn't catch everything, but i was trying to address a TSLint warning that didn't like how the Promise above was nested within another Promise that both used resolve/reject as their parameter names. i'll revert this change though if you feel like it's a bit unnecessary

Reply:
ohhh!!! got it! nope, leave it as is. makes sense. thanks for clarifying.